lib/presentation/widget/upload_file/upload_file_arguments.dart

```diff
@@ -44,3 +44,5 @@ class UploadFileArguments {
 }
 
 enum ShareType { quickShare, uploadAndShare }
+
+enum ShareButtonType { justUpload, uploadAndShare }
```

Comment:
two have the same intent. I bet

Reply:
Ive checked code. `ShareType` intents there are two type of screen. One for just sharing file, and one for Upload and Share.

Then `ShareButtonType` just for the second screen type "Upload and Share". And it's depend on number of sharing recipients.

The intent of two enum class is different, plz consider

Reply:
okie to me